content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -175,11 +165,11 @@ export function getInfoItems(exhibition: UiExhibition) {
   ].filter(Boolean);
 }
 
-type Props = {|
-  exhibition: UiExhibition,
-  pages: Page[],
-  globalContextData: any,
-|};
+type Props = {
+  exhibition: UiExhibition;
+  pages: Page[];
+  globalContextData: any;
```

Comment:
I got yelled at when I tried this; I think there's a GlobalContextData type you can use here? Or a WithGlobalContextData you can && onto the Props.

Reply:
True, jstu fixed it.